action.yml

```diff
@@ -29,27 +29,60 @@ outputs:
 runs:
   using: composite
   steps:
+    - id: registryconfig
+      if: ${{ cloudbees.registries != '' }}
+      name: Resolve source image
+      uses: docker://020229604682.dkr.ecr.us-east-1.amazonaws.com/services/registry-config:0.0.22
+      with:
+        entrypoint: /registry-config
+        args: --config="${{ cloudbees.registries }}" resolve "${{ inputs.src }}" resolved-image.file
```

Comment:
Can we expose the registries config file as an Action input that defaults to `${{cloudbees.registries}}`?
That is to a) clarify for users that/how source image references are rewritten, taking the configured registry mirrors into account and b) allow them to overwrite the registry mirror configuration for a specific Action call in case they really need to.